wps-common/src/main/resources/xml/DescribeProcess/GoGoDuck.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wps:ProcessDescriptions xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:wps="http://www.opengis.net/wps/1.0.0"
```

Comment:
Can we just include the ProcessDescription element?  The ProcessDescriptions elelemtn is superfluous.

Reply:
I tried that exact method - but got XML unmarshalling issues - so went down this path.